autopush-common/src/db/bigtable/bigtable_client/mod.rs

```diff
@@ -1442,5 +1454,24 @@ mod tests {
         let err = client.add_user(&user).await.unwrap_err();
         assert!(matches!(err, DbError::Conditional));
     }
-    */
+
+    #[actix_rt::test]
+    async fn version_check() {
+        let client = new_client().unwrap();
+        let uaid = Uuid::new_v4();
+        let user = User {
+            uaid,
+            ..Default::default()
+        };
+        client.remove_user(&uaid).await.unwrap();
```

Comment:
We definitely want to use a "safer" value here. 

probably also should clean it up at the end of the test.

Reply:
I believe you had some code to randomize these uuids with a prefix somewhere, if you point me to it I'll add it to this branch to resolve both comments

Reply:
https://github.com/mozilla-services/autopush-rs/pull/588/files#diff-773a699c0731038f9168c5bb9ffe0ce0aad1f5d05c835bdd6edfe20f2dfa050fR1

Reply:
I thought it was on some WIP/draft branch, no of course it's in a PR I've been ignoring 🤦‍♂️